packages/collector/test/tracing/common/test.js

```diff
@@ -168,23 +168,6 @@ mochaSuiteFn('tracing/common', function () {
       );
     });
 
-    describe('with header when agent is configured to capture the header', function () {
-      const agentControls = new AgentStubControls();
-      const agentConfig = { extraHeaders: ['x-iNsTanA-sErViCe'] };
```

Comment:
qs: Should we keep the test but just use a different header?

Reply:
The test was expecting the service name from the headers, but since we removed the implementation, the test can no longer be reused.